File: SPSurvey.Core/SDK/QuestionItem.vb
Imports ControlOrigins.COUtility

Public Class QuestionItem
    Public Property QuestionID As Integer
    Public Property QuestionSort As Integer
    Public Property QuestionNM As String
    Public Property QuestionShortNM As String
    Public Property QuestionTypeCD As String
    Public Property QuestionTypeID As Integer
    Public Property ControlNM As String
    Public Property AnswerDataType As String
    Public Property CommentFL As Boolean
    Public Property MaxQuestionValue As Integer
    Public Property QuestionDS As String
    Public Property QuestionValue As Decimal
    Public Property ReviewRoleLevel As Integer
    Public Property SurveyTypeID As Integer
    Public Property UnitOfMeasureID As Integer
    Public Property FileData As Byte()
    Public Property ModifiedID As Integer
    Public Property Keywords As String
    '
    ' Possible Answers for this Question
    '
    Public Property QuestionAnswerItemList As New List(Of QuestionAnswerItem)
    '
    '  Only Implemented when Question is in a QuestionGroup or SurveyQuestionList
    Public Property QuestionGroupMember As New QuestionGroupMemberItem
    '  Only Implemented when Question is in a SurveyResponse.Survey.Question 
    Public Property SurveyResponseAnswerItemList As New List(Of SurveyResponseAnswerItem)
    Public Property SurveyDisplayOrder As Integer
    Public Property SurveyLookupList As New List(Of LookupItem)
End Class

File: ControlOrigins.Survey.Common/SDK/QuestionItem.cs
using System.Collections.Generic;

namespace ControlOrigins.Survey.Common.SDK
{
    public class QuestionItem
    {
        public int QuestionID { get; set; }
        public int QuestionSort { get; set; }
        public string QuestionNM { get; set; }
        public string QuestionShortNM { get; set; }
        public string QuestionTypeCD { get; set; }
        public int QuestionTypeID { get; set; }
        public string ControlNM { get; set; }
        public string AnswerDataType { get; set; }
        public bool CommentFL { get; set; }
        public int MaxQuestionValue { get; set; }
        public string QuestionDS { get; set; }
        public decimal QuestionValue { get; set; }
        public int ReviewRoleLevel { get; set; }
        public int SurveyTypeID { get; set; }
        public int UnitOfMeasureID { get; set; }
        public byte[] FileData { get; set; }
        public int ModifiedID { get; set; }
        public string Keywords { get; set; }
        // 
        // Possible Answers for this Question
        // 
        public List<QuestionAnswerItem> QuestionAnswerItemList { get; set; } = new List<QuestionAnswerItem>();
        // 
        // Only Implemented when Question is in a QuestionGroup or SurveyQuestionList
        public QuestionGroupMemberItem QuestionGroupMember { get; set; } = new QuestionGroupMemberItem();
        // Only Implemented when Question is in a SurveyResponse.Survey.Question 
        public List<SurveyResponseAnswerItem> SurveyResponseAnswerItemList { get; set; } = new List<SurveyResponseAnswerItem>();
        public int SurveyDisplayOrder { get; set; }
    }
}